_includes/footer.html

```diff
@@ -54,6 +54,14 @@
             class="brand-icon"
           />
         </span>
+      <a class="navbar-item" href="https://vimeo.com/solidworld" title="Solid on Vimeo">
```

Comment:
To avoid breaking the  Twitter link:

```suggestion
      </a>
      <a class="navbar-item" href="https://vimeo.com/solidworld" title="Solid on Vimeo">
```